readme.md

```diff
@@ -138,6 +139,22 @@ These shims allow you to use systems with presentation metadata (e.g. structure
 - [iiif-producer](https://github.com/ubleipzig/iiif-producer) - A CLI tool that generates IIIF Presentation 2.1 Manifests from METS/MODS (produced by Kitodo).
 - [File Analyzer IIIF Manifest Generator from Existing Metadata](http://georgetown-university-libraries.github.io/File-Analyzer-Test-Data/iiif/) - Georgetown's desktop application that will generate IIIF manifest files from existing metadata files.
 
+## Exhibition and Guided Viewing Tools
+
+Tools and resources that provide functionality for presenting IIIF materials in an exhibition-like setting (and potentially other functionality).
+
+- [Exhibit](https://exhibit.so/) - A free IIIF storytelling tool that allows for guided navigation of one or more IIIF Manifests using annotations.  
+[Storiiies Editor](https://storiiies-editor.cogapp.com/) – a free online storytelling platform for creating guided tours of a single IIIF manifest using annotations.
```

Comment:
Should there be a dash for this to be its own bullet point?

Reply:
Yes, great catch!